src/tests/ftest/rebuild/basic.yaml

```diff
@@ -2,9 +2,10 @@
 # required quantity is indicated by the placeholders
 hosts:
   test_servers: 6
-timeout: 900
+timeout: 600
```

Comment:
Each variant across all VM OSes completed in less than 5 minutes
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14398/2/testReport/FTEST_rebuild/RbldBasic/
Setting this to 10 minutes to account for rebuild variability